superset/assets/javascripts/explore/stores/store.js

```diff
@@ -32,6 +32,11 @@ export function getControlsState(state, form_data) {
   const formData = Object.assign({}, form_data);
   const vizType = formData.viz_type || 'table';
 
+  // Control reacffectation for deprecation
```

Comment:
creating `function handleDeprecatedControls(formData)` to clarify and segment things